client/src/hooks/useApiCallback.js

```diff
@@ -44,9 +45,15 @@ const useApiCallback = (method, url, initialData) => {
             dispatch({ type: types.FETCH_SUCCESS, payload: data });
           }
         } catch (error) {
+          console.log("error", error.response.data.message)
```

Comment:
We can remove this console log as the error message is being provided in state if we want to inspect it